setup/baseConfig.yaml

```diff
@@ -105,6 +105,8 @@ azstorage:
   sdk-trace: true|false <enable storage sdk logging>
   fail-unsupported-op: true|false <for block blob account return failure for unsupported operations like chmod and chown>
   auth-resource: <resource string to be used during OAuth token retrieval>
+  update-md5: true|false <set md5 sum on upload. Impacts performance>
```

Comment:
This is only possible with file cache right? Should we add that as a note here?

Reply:
True, will add comment here.